### PR TITLE
Picarto Fixed

### LINKED ISSUE #3384: Picarto | picarto.tv
**Describe the Bug**
Container only says "Browsing..", does not show streams that are playing or play/paused status.

**Steps to Reproduce**
1. Go to picarto.tv
2. Open any stream

**Expected Behavior**
Expected to see the name of the streamer I'm watching in Discord RPC, however it's either only saying "Browsing.." or isn't picking up the page at all.

**Environment**
- OS: Windows 10 Pro 20H2
- Browser: Vivaldi
- Version: 3.6.2165.40 (Stable channel) (64-bit)
- PreMiD Extension Version: 2.2.0.7 beta
- PreMiD App Version: 2.2.0

**Additional context**
Picarto appears to have recieved a major update recently, so I'm not surprised it's not working as expected, but figured it would be good to let the devs of the container know.

### PR DESCRIPTION
**Pull request by DarkVillager**
- Picarto metadata updated,
- Picarto fully works again.
image
image

### CODE REVIEW

File: websites/P/Picarto/dist/metadata.json [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-  "thumbnail": "https://i.imgur.com/DSNnMk5.png",
-  "color": "#FF0000",
-  "tags": [
-    "video"
-  ],
-  "category": "videos"
-}
\ No newline at end of file
+  }
+}
```
Comment (Bas950) [inline websites/P/Picarto/dist/metadata.json]: there is no need to change the order...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/Picarto/dist/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/P/Picarto/dist/metadata.json]: There kinda is, it broke for me without the old metadata?

Comment (Bas950) [reply websites/P/Picarto/dist/metadata.json]: bruh it's json... it doesn't matter.
You still just do metadata.author etc.

**Comment by EncryptedDev** [comment]
🤨
> If a pull request is inactive for a period of 7 days, it will be promptly closed.

**Pull request closed without merging**